portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplayWithStaging.testcase

```diff
@@ -101,4 +101,92 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-127548. View the selected web content just is shown after Published to live."
```

Comment:
The changes of web content won't be shown in Content Display in live site until publish to live